WORKSPACE

```diff
@@ -119,6 +119,30 @@ go_repository(
     version = "v0.6.0",
 )
 
+go_repository(
+    name = "googleapis",
```

Comment:
Shouldn't we name this `com_github_googleapis_googleapis` now that it isn't subject to any special handling anymore?

Reply:
This repo will be treated as a pure proto repo. All references to it will have to be resolved manually (possibly with the new `resolve_regexp` directive). Because of that, I am trying to save some typing...

Do you see any benefit for using `com_github_googleapis_googleapis`?

Reply:
I see. I was just wondering what this would look like with Bzlmod: If the repo is managed by `go_deps`, it would naturally be assigned this name unless we special case it. Having WORKSPACE and Bzlmod builds align on the directives users need to add could have sufficient value that the longer repo name becomes acceptable.

Reply:
googleapis is not going to be managed by `go_deps`, because the Go modules are in `org_golang_google_genproto`. The only reason to download it as a `go_repository` is for Gazelle to generate the `proto_library` rules, hence `gazelle:go_generate_proto false` and `build_file_generation = "on"`.

Reply:
I see thanks for the explanation!